src/components/frame/LandingPage/InfoBanner/InfoBanner.tsx

```diff
@@ -0,0 +1,82 @@
+import { type Icon } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React from 'react';
+
+import { formatText } from '~utils/intl.ts';
+
+import LoadingInfoBanner from './LoadingInfoBanner.tsx';
+
+export interface InfoBannerProps {
+  variant?: 'info' | 'success' | 'error';
+  icon?: Icon;
+  loading?: boolean;
+  title: string;
+  text: string;
+}
+
+const displayName = 'frame.LandingPage.InfoBanner';
+
+const InfoBanner = ({
+  variant = 'info',
+  icon: Icon,
+  loading,
+  title,
+  text,
+}: InfoBannerProps) => {
+  const containerClassNames = 'rounded-lg border p-6';
+  const contentClassNames = 'flex items-center gap-1.5 pb-2 pt-3';
```

Comment:
Can you use the tw utility function here so that classes are properly sorted?

```
import { tw } from '~utils/css/index.ts';

const containerClassNames = tw`rounded-lg border p-6`;
const contentClassNames = tw`flex items-center gap-1.5 pb-2 pt-3`;
```